pkg/gen/input/renovate/internal/file/renovate.json.template

```diff
@@ -4,6 +4,9 @@
 {
   "extends": [
     "config:base"
+    {{- if $reviewer }},
+    ":reviewer({{ $reviewer | printf "team:%s" }})"
```

Comment:
I think using extends for reviewers is more cleaner.

Reply:
How is reviewer set if there is no flag?

Reply:
It would assign CODEOWNER as a reviewer automatically. I think it makes sense to loose the restriction on the reviewer flag when it has been not working properly from the renovate side.